asset/gulpfile.js

```diff
@@ -0,0 +1,77 @@
+require('es6-promise').polyfill();
+
+var gulp = require('gulp'),
+    cssimport = require('gulp-cssimport'),
+    sass = require('gulp-sass'),
+    concat = require('gulp-concat'),
+    uglify = require('gulp-uglify'),
+    rev = require('gulp-rev'),
+    prism = [
+        'core',
+        'markup',
+        'css',
+        'clike',
+        'javascript',
+        'apacheconf',
+        'bashv',
```

Comment:
Is this correct? Shouldn't it be just "bash"? (See https://github.com/PrismJS/prism/blob/gh-pages/components/prism-bash.js)